keyboards/xd75/keymaps/default/readme.md

```diff
@@ -1 +1,8 @@
-# The default keymap for xd75, with led controls
+# Boy_314's XD75RE Layout
```

Comment:
Could you revert this?

Reply:
Do you want me to replace my entire readme with the default one? Sorry I'm new to this and the formatting is kinda confusing to me.

Reply:
Oh my bad I'm looking at the wrong file.